### PR TITLE
Allow for multiple logos

### PR DESCRIPTION
**Pull request by Rouxles**

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF). The following exceptions apply:
         - 3k2a) If a non-permitted puzzle is found before a round is complete, affected results in the round may be replaced with extra attempts, at the discretion of the WCA Delegate.
         - 3k2b) 3x3x3 Multi-Blind: if any puzzles are found to be non-permitted, such puzzles may be individually counted as unsolved (without disqualifying the entire attempt), at the discretion of the WCA Delegate.
-- 3l) A puzzle may have a logo on a colored part. If it does, it must have at most one colored part with a logo. Exception: For blindfolded events, a puzzle must not have a logo.
+- 3l) A puzzle may have a logo or multiple logos on a colored part. If it does, it must have at most one colored part with a logo. Exception: For blindfolded events, a puzzle must not have a logo.
```
Comment (Nanush7) [inline wca-regulations.md]:
```suggestion
- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with logos. Exception: For blindfolded events, a puzzle must not have any logos.
```

Comment (ohexter) [reply wca-regulations.md]:
```suggestion
- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with logos. Exception: For blindfolded events, a puzzle must not have logos.
```
Alternatively "a logo or logos", although it looks like the plural can refer to the singular elsewhere in the regulations (e.g. E3c, 5a), so this should be sufficient).

Comment (Nevseros) [inline wca-regulations.md]:
```suggestion
- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with a logo or multiple logos. Exception: For blindfolded events, a puzzle must not have any logos.
```

**Comment by Nanush7** [comment]
Which suggestion are we going to use?

**Comment by Nevseros** [comment]
The last suggestion is the one we decided to use in the meeting.

File: wca-regulations.md (hunk 1 above)
Comment (OJCubing) [reply wca-regulations.md]: I'm still in favour of "any logos" for the last line (BLD). "must not have logos" sounds more odd to me.

**Review by Nevseros — APPROVED** [review]
LGTM!

**Pull request merged**